README.md

```diff
@@ -5,11 +5,38 @@ Create SaaS Boilerplate
 [![Downloads/week](https://img.shields.io/npm/dw/create-saas-boilerplate.svg?style=for-the-badge)](https://npmjs.org/package/create-saas-boilerplate)
 [![License](https://img.shields.io/npm/l/create-saas-boilerplate.svg?style=for-the-badge)](https://github.com/apptension/create-saas-boilerplate/blob/master/package.json)
 
-<!-- toc -->
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
-# Usage
+Create SaaS Boilerplate is a CLI tool designed to quickly set up a new local instance of the SaaS Boilerplate. It is a specialized tool that allows users to get started with the SaaS Boilerplate as quickly as possible. 
+
+---
+
+📖 [**SaaS Boilerplate Documentation**](https://docs.demo.saas.apptoku.com/)
+
+🌟 [**SaaS Boilerplate Page**](https://www.apptension.com/saas-boilerplate)
+
+---
+
+## Initialize new SaaS Boilerplate project
+
+```
+USAGE
+  $ yarn create saas-boilerplate PATH [--skipSystemCheck]
+  $ pnpm create saas-boilerplate PATH [--skipSystemCheck]
+  $ npm init saas-boilerplate PATH [--skipSystemCheck]
+ARGUMENTS
+  PATH  [default: .] Directory name where to initialize project
+FLAGS
+  --skipSystemCheck  Skip system check
+DESCRIPTION
+  Initialize new SaaS Boilerplate project
+```
+_See code: [dist/commands/init/index.ts](https://github.com/apptension/create-saas-boilerplate/blob/v0.0.1/dist/commands/init/index.ts)_
```

Comment:
I don't think we need this link tbh. If possible I would remove "See code" links

Reply:
I can remove it from here and it will be fine. But in the `Commands` section, they'll be auto-generated by `oclif-readme` anyways

Reply:
I know! ;)